src/main/java/teammates/storage/api/CommentsDb.java

```diff
@@ -340,6 +341,21 @@ private void updateInstructorEmailAsGiver(String courseId, String oldInstrEmail,
         getPm().close();
     }
     
+    /*
+     * Updates last editor (an instructor email) with the new one
+     */
+    private void updateInstructorEmailAsLastEditor(String courseId, String oldInstrEmail,
+                                                   String updatedInstrEmail) {
```

Comment:
This method signature doesn't make sense. Update the last editor of which comments? Each method should make sense on its own.